### DISCUSSION TITLE
ansible-practice

### DISCUSSION THREAD

**Discussion by patelvamshidhar** [learndevopsonline/learndevopsonline#1876] [Q&A]
while connection to the gocd with the GitHub i am getting below error
![image](https://github.com/learndevopsonline/learndevopsonline/assets/49507145/b83ac341-e1bf-4748-b703-9487055c8bcb)

**Comment by chaithanya1812** [comment]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/d1038657-2997-42d3-99c9-b655e30bee87)

**Reply by patelvamshidhar** [reply]
I have removed the directory 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/49507145/b469a977-437a-4f99-88d1-b56768b52cb9)

**Comment by chaithanya1812** [comment]
Hi @patelvamshi ,
share you GoCD public-ip

**Comment by patelvamshidhar** [comment]
44.203.143.207 | 172.31.85.199--- Godcd server
34.237.223.124 | 172.31.3.190 -- gocd agent

**Comment by chaithanya1812** [comment]
hi @patelvamshi, I just added the repo, we are getting some errors, so will you please join lab session once?
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/7d1bb927-e1b5-4c25-8640-83fb4ba64317)

**Reply by patelvamshidhar** [reply]
Okay bro thankyou @chaithanya1812
